packages/terra-tabs/tests/wdio/tabs-spec.js

```diff
@@ -15,26 +15,26 @@ describe('Tabs - Responsive', () => {
         browser.setViewportSize(viewport);
         browser.moveToObject('.tabContent');
       });
-      Terra.should.matchScreenshot({ viewports });
-      Terra.should.beAccessible({ viewports });
+      Terra.should.matchScreenshot({ viewports: [viewport] });
```

Comment:
You should be able reduce this to Terra.should.matchScreenshot();

Reply:
Thanks! Removed it, and it works great!

https://github.com/cerner/terra-core/pull/1194/commits/799ad0e50a2c0280d130342f43be767f9d020549